deployments/extensions.go

```diff
@@ -0,0 +1,405 @@
+}
+
+func (e *Extension) executeScript(path string) error {
+	tmpDir, err := ioutil.TempDir("", "fuseml-extension-chart")
+	if err != nil {
+		return errors.Wrap(err, "can't create temp directory "+tmpDir)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	fullCmd, err := e.fetchFile(path, tmpDir)
+	if err != nil {
+		return errors.Wrap(err, "failed fetching file from "+path)
+	}
+
+	if out, err := helpers.RunProc(fullCmd, tmpDir, e.Debug); err != nil {
+		return errors.Wrap(err, fmt.Sprintf("Failed running script: %s\n", out))
+	}
+
+	return nil
+}
+
+func (e *Extension) installManifest(path, ns string) error {
+	tmpDir, err := ioutil.TempDir("", "fuseml-extension-chart")
+	if err != nil {
+		return errors.Wrap(err, "can't create temp directory "+tmpDir)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	manifestLocalPath, err := e.fetchFile(path, tmpDir)
+	if err != nil {
+		return errors.Wrap(err, "failed fetching file from "+path)
+	}
+
+	out, err := helpers.Kubectl(fmt.Sprintf("apply --filename %s --namespace %s", manifestLocalPath, ns))
+
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("kubectl apply failed:\n%s", out))
+	}
+	return nil
+}
+
+func (e *Extension) unInstallManifest(path, ns string) error {
+	tmpDir, err := ioutil.TempDir("", "fuseml-extension-chart")
+	if err != nil {
+		return errors.Wrap(err, "can't create temp directory "+tmpDir)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	manifestLocalPath, err := e.fetchFile(path, tmpDir)
+	if err != nil {
+		return errors.Wrap(err, "failed fetching file from "+path)
+	}
+
+	out, err := helpers.Kubectl(fmt.Sprintf("delete --filename %s --namespace %s", manifestLocalPath, ns))
+
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("kubectl delete failed:\n%s", out))
+	}
+	return nil
+}
+
+// TODO move under helpers
+func (e *Extension) installHelmPackage(name, chartPath, ns, valuesPath string) error {
+
+	tmpDir, err := ioutil.TempDir("", "fuseml-extension-chart")
+	if err != nil {
+		return errors.Wrap(err, "can't create temp directory "+tmpDir)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	tarName := filepath.Base(chartPath)
+	if err = helpers.DownloadFile(chartPath, tarName, tmpDir); err != nil {
+		return errors.Wrap(err, "can't download helm chart for "+name)
+	}
+
+	chartLocalPath := filepath.Join(tmpDir, tarName)
+	valuesLocalPath := ""
+
+	if valuesPath != "" {
+		valuesLocalPath, err = e.fetchFile(valuesPath, tmpDir)
+		if err != nil {
+			return errors.Wrap(err, "failed fetching file from "+valuesPath)
+		}
+	}
+
+	helmCmd := fmt.Sprintf("helm install %s --create-namespace --values '%s' --namespace %s --wait %s", name, valuesLocalPath, ns, chartLocalPath)
+	currentdir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	if out, err := helpers.RunProc(helmCmd, currentdir, e.Debug); err != nil {
+		return errors.New(fmt.Sprintf("Failed installing %s chart: %s", name, out))
+	}
+
+	return nil
+}
+
+func (e *Extension) unInstallHelmPackage(ui *ui.UI, name, ns string) error {
```

Comment:
same here, for using `HelmChart` instead of `HelmPackage`

Reply:
will do